src/navigators/BottomNavigator.tsx

```diff
@@ -22,6 +22,11 @@ const BottomNavigator = () => {
     showLabelsInNav = false,
   } = useAppSettings();
 
+  const { filteredInstalledPlugins } = usePlugins();
+  const pluginsWithUpdate = filteredInstalledPlugins.filter(
+    p => p.hasUpdate,
+  ).length;
```

Comment:
useMemo has dependencies option, 
```js
const pluginsWithUpdate = useMemo(() => filteredInstalledPlugins.filter(
    p => p.hasUpdate,
  ).length;, [filteredInstalledPlugins]);
```

Reply:
i guess its fine cus when u update a plugin it creates a new plugin object instead of modifying existing, when u refresh the plugin indicator it just mutates existing objects tho so memo wont be rerun

tbh users arent likely to manually reload plugins tho so ig fine